### ISSUE TITLE
[Request]: make deleted vehicle remove all its attachments

### ISSUE THREAD

**Issue by alyxme** [Yimura/YimMenu#1659]
### Problem

When persistent or xml vehicle removed the main driveable vehicle dissapear as intended but then all attachment stays in the world and become just invisible with all their collision remain. That cause unwanted collisions with invisible (ex-attached) objects, npcs are bump into it like a fly at the window, players also crashes into it.

### Solution

Removed vehicle should delete all its attachments/peds.

### Reason

To make gameworld clean after removing for yourself unused vehicles.

**Comment by Tayipc38** [comment]
Could you try using clear entities features located in te Nearby tab?

**Comment by V1RU513N** [comment]
> Could you try using clear entities features located in te Nearby tab?

With this option some entities are removed, others just become invisible
Edit: add video & others
Edit2: correcting information
<details>
<summary>Video</summary>

https://github.com/YimMenu/YimMenu/assets/5224766/b5a3b9ea-af73-4ddb-a7ec-608c88eb8535

</details>
<details>
<summary>Off topic :
Other things about xml vehicles</summary>
Here's the difference between what I see and what the other players see

![GTA5_2023-07-11_19-34-44](https://github.com/YimMenu/YimMenu/assets/5224766/d8df4bae-33a5-49d8-8b2d-adabb2bf2a04)

Some vehicles are not fully generated, and some elements are missing.(menyoo VS yim)

![2023-07-11_19-46-311](https://github.com/YimMenu/YimMenu/assets/5224766/b0e6bed7-b23d-4c87-9c06-6da08e8c97a3)


Other things (I don't know if it's really a bug) particles are only visible locally.

![2023-07-11_18-04-38](https://github.com/YimMenu/YimMenu/assets/5224766/08075b96-dee7-46d6-817d-1652bbe562d0)

</details>

**Comment by alyxme** [comment]
> Could you try using clear entities features located in te Nearby tab?

Yes it could clean that but if other players with their Vehicles are also nearby their vehs may also can be affected to deletion and their Reaction settings may backfire causing more troubles. So proper removing is preferable if it possible in simple way. Also somehow these attachments usualy won't be cleared with nearby function or with manual pointing to delete.

Edit: Is it ok to set to all attachments collision:disable before deleting main vehicle?! So they just fall into the void. Since they are set to invisible when deleted it would be the solution to set their collision to disable simultaneously with invis. So messed attachments don't bother again after.
